eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -148,15 +148,23 @@ stages:
     - job: Analyze
       displayName: Analyze
       variables:
-          Codeql.Enabled: true
-          Codeql.BuildIdentifier: ${{ parameters.ServiceDirectory }}
-          Codeql.SkipTaskAutoInjection: false
+        - template: /eng/pipelines/templates/variables/globals.yml
```

Comment:
Actually given this is imported at the stage level above isn't it already included?

Reply:
Oh you're right I was thinking it was imported at the job scope not the stage scope.